src/core/QCheck.mli

```diff
@@ -826,6 +843,12 @@ module TestResult : sig
         @since 0.9 *)
   }
 
+  val get_count : _ t -> int
```

Comment:
```suggestion
  val get_count : _ t -> int
  
 (** Get the count of a cell.
     @since 0.5.3 *)
```

Reply:
Done in faf202c706bd6b5439a10cb9c5dc10393b2fd33e